addons/website_event_jitsi/views/res_config_settings_views.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="res_config_settings_view_form" model="ir.ui.view">
-        <field name="name">res.config.settings.view.form.inherit.website.jitsi</field>
+        <field name="name">res.config.settings.view.form.inherit.event</field>
         <field name="model">res.config.settings</field>
-        <field name="inherit_id" ref="website.res_config_settings_view_form" />
+        <field name="inherit_id" ref="event.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <xpath expr="//div[@id='google_maps_setting']" position="after">
+            <xpath expr="//div[@name='events_setting_container']" position="inside">
```

Comment:
Using inside leads to a somehow undefined ordering depending on module installation order. Doesn't that break anything ? Could be solved by using priorities.

Reply:
In this case, there's only one override.

But even if there were many of them, this is not a big deal in our case. The goal is to group the settings by thematic, and there are generally between 1 to 8 settings by section.